src/components/PostInfo/PostInfo.jsx

```diff
@@ -1,3 +1,25 @@
-export const PostInfo = () => (
-  <>Put the post here</>
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+import './PostInfo.scss';
+
+export const PostInfo = ({ post: { title, user, body, comments } }) => (
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">{title}</h3>
+
+      <p>
+        {' Posted by  '}
+
+        <UserInfo user={user} />
+      </p>
+    </div>
+
+    <p className="PostInfo__body">{body}</p>
+
+    <hr />
+
+    {comments.length === 0
```

Comment:
here you can just leave  `{comments.length ? (`
this value will be automatically converted to boolean, if it would be 0 then it would be false, if a positive number then true